### ISSUE TITLE
Tags confidence

### ISSUE THREAD

**Issue by grafael** [guillaumegenthial/sequence_tagging#3]
It is possible to get the tag confidence or "**likelihood**" ? 
I mean, in the sentence: _I love Paris_

| word | tag |confidence|
|------|-----|-----------|
 I | O| 99%
love|O|75%
Paris|B-PLACE|89%

**Comment by guillaumegenthial** [comment]
Hello Rafael,

Without the CRF, you can get this confidence (not normalized, you would have to add the softmax) with this line

self.logits = tf.reshape(pred, [-1, ntime_steps, self.ntags])

which is defined in model.py.

For the CRF, you can also use this quantity (which are emissions scores), but as the CRF has transitions scores, this may be not exactly what you want. (but you could find a way to combine both).

I hope this helps,

Best,

Guillaume

**Comment by grafael** [comment]
Thanks @guillaumegenthial I'll try that.
